### DISCUSSION TITLE
Grouped Inline Actions

### DISCUSSION THREAD

**Discussion by thethunderturner** [filamentphp/filament#15028] [Ideas]
Can we have grouped inline header actions? Similar to toggle buttons
![image](https://github.com/user-attachments/assets/7227c5ba-0f95-487f-b12b-9595fa62fa2f)

**Comment by thethunderturner** [comment]
@danharrin Do you think this is a good feature? If so, I can make a PR for it.

**Comment by danharrin** [comment]
Hi, thanks for asking but not had enough demand to implement this yet, sorry.
